cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/cube/CubeDeleteQuery.java

```diff
@@ -68,8 +78,9 @@ public String toString() {
     return Objects.toStringHelper(this)
       .add("startTs", startTs)
       .add("endTs", endTs)
+      .add("resolution", resolution)
       .add("measureName", measureName)
-      .add("sliceByTags", Joiner.on(",").withKeyValueSeparator(":").useForNull("null").join(sliceByTagValues))
+      .add("sliceByTags", Joiner.on(",").withKeyValueSeparator(":").join(sliceByTagValues))
```

Comment:
No guava class in cdap-api module (we are trying to get rid of the dependency). Simple add the map and relies on the Map.toString.